modules/claims_api/app/workers/claims_api/encrypt_pii.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class EncryptPii
```

Comment:
we're no longer encrypting and there is no PII here. I would change this class name to something more meaningful. Maybe something like `ClaimsApi::UpdaterService`?